python/cuml/feature_extraction/_tfidf_vectorizer.py

```diff
@@ -0,0 +1,253 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Portions of this code are derived from the scikit-learn feature_extraction
+# package, which has the following license:
+#
+#    -*- coding: utf-8 -*-
+#    Authors: Olivier Grisel <olivier.grisel@ensta.org>
+#             Mathieu Blondel <mathieu@mblondel.org>
+#             Lars Buitinck
+#             Robert Layton <robertlayton@gmail.com>
+#             Jochen Wersdörfer <jochen@wersdoerfer.de>
+#             Roman Sinayev <roman.sinayev@gmail.com>
+#
+#    License: BSD 3 clause
+#
```

Comment:
Ouch, I forgot that sklearn doesn't include the full license in their header. To comply with BSD, we need a full copy of that license in this codebase as well. Can you add a file called `cuml/thirdparty/LICENSES/LICENSE.scikit_learn` that exactly reproduces the sklearn license: `https://github.com/scikit-learn/scikit-learn/blob/master/COPYING`

Reply:
Got it, thanks!